model/Core/Properties/created.md

```diff
@@ -9,7 +9,7 @@ Identifies when the Element was originally created.
 ## Description
 
 Created is a date that identifies when the Element was originally created.
-The time stamp can serve as an indication as to whether the analysis needs to be updated.
+The time stamp can serve as an indication as to whether the analysis needs to be updated. This is often the date of last change (e.g., a git commit date), not the date when the SPDX data was created, as doing so supports reproducible builds.
 
```

Comment:
Same comment as above - similar wording suggested.

Reply:
That wording looks good to me. It explains what it's *for* not just what it *is*, and I suspect that helps.

Reply:
@rpurdie - what do you think?

Reply:
It is fine with me, I'm happy if we can have reproducible builds and the commit date/source epoch gives us that.